torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -2816,49 +2859,75 @@ void checkSortSchema(const Node* node, const c10::TypePtr& list_element_type) {
               << class_type->python_str() << " that "
               << "returns a bool";
   } else {
-    error_str
-        << "Input to list sort must be of Tensors, ints, floats, bools or "
-        << "a User Defined Class that defines the __lt__ compare method"
-        << ", got list of " << list_element_type->python_str() << "\n";
+    error_str << "Input to " << node->kind().toUnqualString()
+              << "must be of Tensors, ints, floats, bools or "
+              << "a User Defined Class that defines the __lt__ compare method"
+              << ", got list of " << list_element_type->python_str() << "\n";
   }
 
   auto error_msg = script::ErrorReport(node->sourceRange());
   error_msg << error_str.str();
   throw error_msg;
 }
 
+Operation sort_op(
+    Function* lt_func,
+    bool has_reverse_arg,
+    bool copy_return_list) {
+  return [lt_func, has_reverse_arg, copy_return_list](Stack& stack) {
+    bool reverse = has_reverse_arg ? pop(stack).toBool() : false;
```

Comment:
The list of `bool` args for `sort_op` is confusing, this could be handled when the actual op is returned (i.e. [here](https://github.com/pytorch/pytorch/pull/23274/files#diff-c3414445c5446fb0bd90895818533319R2918)) instead of here which would clean things up.

Reply:
i'm not sure what you mean